tests/test_integration.py

```diff
@@ -1022,10 +1022,12 @@ def pex_with_entrypoints(entry_point):
   """)
 
   my_app = dedent("""
-    from setuptools.sandbox import run_setup
-
     def do_something():
-      return run_setup
+      try:
+        from setuptools.sandbox import run_setup
+        return 0
+      except:
+        return 1
 
     if __name__ == '__main__':
       do_something()
```

Comment:
You beat me to it. As you see fit: https://github.com/pantsbuild/pex/pull/606
The do_something() should probably be used to sys.exit.